environments/pull-environment.sh

```diff
@@ -2,8 +2,12 @@
 set -e
```

Comment:
Note that this is the child helper script that `pull-environment*s*.sh` uses. This file doesn't have code updates, other than accepting one more argument ("region").